package.json

```diff
@@ -14,6 +14,7 @@
     "react-map-gl": "^7.1.5"
   },
   "devDependencies": {
+    "@jupyter-widgets/base": "^6.0.6",
```

Comment:
This is a type-only dependency, so I think it's ok to have it as a dev dependency. And probably better this way, as it won't interact with whatever version jupyterlab is using.